### PR TITLE
Build: Fix the sb-bench CI step

### PR DESCRIPTION
**Pull request by ndelangen**
Issue: The CI task of sb-bench was failing.

- I had to update and sync up the dependencies of playwright with the ones in the image used for the CI runner.
- I had to fix some issues after the emotion upgrade because I had to regenerate the yarn lockfile to do the above.

related: #18992 I think this replaces it. @Andarist

### CODE REVIEW

**Review by shilman — APPROVED** [review]
You are the king!!! 👑

File: scripts/prepare/bundle.ts [hunk 1]
```diff
@@ -80,6 +80,7 @@ const run = async ({ cwd, flags }: { cwd: string; flags: string[] }) => {
           : false,
       esbuildOptions: (c) => {
         /* eslint-disable no-param-reassign */
+        c.conditions = ['module'];
```
Comment (Andarist) [inline scripts/prepare/bundle.ts:83]: I would recheck what kind of an effect does it have on esbuild - is this purely additive to the default conditions? or does it replace them? I think that I was checking this in the past and that this is additive - some default conditions just cant be turned off but I'm not 100% sure.

**Comment by Andarist** [comment]
> related: https://github.com/storybookjs/storybook/pull/18992 I think this replaces it. @Andarist

Yes, this is same-ish - I would still recommend updating the dependency ranges for Emotion deps in `package.json` files though.

**Comment by ndelangen** [comment]
So it seems there's a bunch of these missing / broken:
https://www.chromatic.com/test?appId=5da9c7ef6e6df60020a7f102&id=6308f03bd7fcbd4674f94f90

**Comment by ndelangen** [comment]
@shilman @tmeasday good to merge?

**Pull request merged**